TYPO3.Flow/Classes/TYPO3/Flow/Security/Cryptography/RsaWalletServicePhp.php

```diff
@@ -41,6 +42,12 @@ class RsaWalletServicePhp implements RsaWalletServiceInterface
      */
     protected $openSSLConfiguration = [];
 
+    /**
+     * The padding to use for OpenSSL encryption/decryption
+     * @var int
+     */
+    protected $paddingAlgorithm = OPENSSL_PKCS1_OAEP_PADDING;
```

Comment:
Does it really make sense to set a default variable here? I find that a bit misleading i it is usually overridden via `injectSettings`. And wouldn't that lead to a PHP error if the constant does not exist (because openssl is not enabled) - or was that a requirement for Flow?

Reply:
Having a default seemed like the right thing. But you are right about the undefined constant. Will fix.